pkgs/development/python-modules/aioprometheus/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, orjson
+, quantile-python
+, aiohttp
+, starlette
+, quart
+, pytestCheckHook
+, asynctest
+, httpx
+, fastapi
+, uvicorn
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "aioprometheus";
+  version = "22.5.0";
+  format = "setuptools";
+
+  disable = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "claws";
+    repo = "aioprometheus";
+    rev = version;
+    hash = "sha256-oXLKJDCJFRcR+yOXpTVuQgrnOfPRnS8g51Aci/qMOOw=";
+  };
+
+  propagatedBuildInputs = [
+    orjson
+    quantile-python
+    aiohttp
+    starlette
+    quart
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    asynctest
```

Comment:
We try to get rid of `asynctest`. Upstream [removed](https://github.com/claws/aioprometheus/blob/master/CHANGELOG.md?plain=1#L23-L24) it for 23.3.0. This is unreleased unfortunately.

It could be worth thinking about going with an `unstable` package of `` for now.

Reply:
Sounds great and tests are not failing on HEAD. Fixed